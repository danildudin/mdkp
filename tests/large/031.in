50 29
384 905 641 308 834 329 35 964 404 421 601 621 675 572 652 253 63 849 933 788 280 165 361 379 426 565 447 443 4 394 22 612 381 630 789 764 674 327 45 976 956 992 295 980 694 337 450 469 355 756 
580 242 314 330 118 587 638 886 39 236 921 582 146 205 678 573 125 359 345 880 857 774 20 566 319 733 383 940 136 505 927 119 580 840 930 878 777 293 137 184 58 822 511 234 841 743 647 56 821 27 41597
66 369 904 264 970 704 607 781 239 789 591 148 549 556 378 919 140 61 819 765 54 647 61 852 615 355 605 186 438 299 506 905 733 228 631 946 715 557 93 673 343 607 469 698 514 136 649 79 741 315 45516
157 265 282 825 480 132 390 76 394 222 818 975 651 705 68 92 611 537 769 949 91 244 818 399 689 778 808 576 430 164 604 82 189 774 300 382 798 944 821 155 155 540 533 620 885 933 591 598 714 938 46216
913 355 394 971 640 165 590 395 723 697 44 843 330 121 248 779 293 54 339 676 740 139 84 246 217 750 147 363 711 294 104 419 607 953 874 564 532 285 569 448 636 757 853 873 365 389 136 956 184 571 86970
98 733 825 530 860 278 640 169 536 538 669 680 21 51 200 48 165 421 65 496 297 471 405 886 710 969 605 423 390 138 525 632 544 620 715 433 309 929 619 191 435 209 881 516 776 481 220 317 170 300 31591
873 970 508 940 705 429 972 122 872 4 390 785 415 937 506 424 68 387 314 493 362 400 180 693 385 391 432 625 53 455 712 619 574 199 894 650 508 6 24 112 895 551 950 984 463 9 317 878 521 759 65481
146 611 885 37 902 203 82 364 120 987 85 407 283 638 62 444 58 389 127 741 634 438 434 439 427 269 78 154 671 555 619 397 985 64 716 922 101 619 458 117 305 50 449 990 502 200 135 172 743 262 36704
469 118 820 122 722 539 716 164 37 865 385 196 283 483 555 277 485 112 817 600 867 167 969 476 723 918 944 145 959 900 511 187 277 306 683 933 184 859 631 439 317 945 620 507 264 623 945 87 763 618 54444
467 822 408 24 765 457 127 906 99 10 752 146 667 219 216 32 458 190 672 107 860 561 644 647 965 687 141 485 519 954 160 944 26 525 108 81 305 338 195 586 522 957 963 454 476 661 128 437 844 817 57914
456 257 700 930 159 364 396 128 897 847 144 890 216 630 672 305 568 620 19 712 981 129 304 260 307 864 413 128 982 992 135 486 155 62 204 785 342 2 525 535 955 815 951 155 442 471 720 894 104 162 15680
285 165 918 843 973 639 98 945 951 658 63 684 786 169 257 875 797 573 33 87 692 723 616 822 296 615 669 998 95 947 811 855 891 675 104 165 661 985 529 773 732 295 583 233 108 198 379 660 693 970 19070
770 579 399 565 702 469 22 652 392 941 64 276 596 5 408 953 963 983 761 963 687 406 146 300 801 334 839 26 33 578 510 115 363 589 29 692 277 134 569 746 368 819 656 619 542 222 307 155 129 198 41605
724 539 327 804 596 141 237 675 340 23 484 110 474 141 997 604 613 214 165 279 845 717 101 376 653 934 636 696 513 351 83 994 891 864 486 456 660 574 223 763 350 475 89 625 329 346 185 623 503 668 4186
974 534 135 509 570 35 425 243 248 385 978 928 247 391 758 779 197 776 403 702 465 804 908 954 813 111 384 98 872 477 195 203 296 219 542 878 275 291 808 184 362 229 257 946 998 117 144 383 608 169 9789
203 741 63 853 972 940 241 963 89 286 543 211 111 868 8 779 653 522 107 306 639 722 1000 640 263 695 341 759 564 516 246 719 691 696 458 496 962 725 80 598 202 47 178 129 194 884 47 683 95 83 59556
55 432 944 23 239 7 599 427 343 330 198 971 630 797 703 438 373 563 795 720 424 276 670 777 329 407 541 877 616 196 227 105 949 721 612 260 335 881 573 359 604 203 939 553 864 56 671 205 611 392 83602
694 379 356 297 38 618 100 865 763 345 516 819 547 681 651 967 532 342 152 252 183 33 787 56 310 623 802 982 808 81 65 267 696 391 214 234 861 452 951 644 843 888 291 896 948 781 333 283 624 836 97928
232 900 324 467 896 696 123 795 779 142 821 692 560 101 624 891 58 70 322 670 873 145 678 344 1 845 913 387 991 721 412 691 95 191 108 389 530 354 662 782 550 71 792 569 276 299 233 589 935 576 38586
913 562 601 373 298 877 966 627 527 456 877 139 49 497 455 43 351 987 593 128 343 907 740 241 7 509 731 811 654 113 621 813 216 576 533 972 656 787 178 886 480 450 984 292 302 116 323 9 30 376 25538
701 436 593 299 472 219 65 234 328 493 481 644 467 870 912 187 794 528 457 633 198 830 792 213 574 663 966 256 664 700 666 365 26 981 544 578 59 200 649 725 879 822 185 421 361 932 148 647 125 426 91047
639 343 741 303 616 978 725 12 528 949 747 682 821 579 437 57 160 826 272 332 458 478 346 917 82 161 116 122 176 494 685 790 722 935 463 925 111 924 663 527 604 1 255 819 450 158 644 459 24 47 80946
431 659 559 663 63 248 999 973 429 406 635 407 330 432 65 575 187 540 636 379 196 290 520 63 825 627 104 682 308 323 311 589 835 567 9 406 143 605 816 257 881 491 18 419 573 984 112 808 650 752 87197
961 486 792 223 444 829 191 174 636 695 941 196 465 323 739 448 3 325 982 571 578 625 567 368 300 680 348 721 373 809 299 800 604 340 982 531 681 746 188 160 879 297 529 376 853 686 135 663 259 488 9305
29 518 177 410 979 609 910 418 756 21 946 349 846 879 359 960 136 476 766 239 849 130 187 966 883 345 473 945 128 201 314 289 644 832 574 505 414 768 670 534 657 430 966 548 804 219 896 655 317 982 29009
426 216 256 395 772 381 977 527 254 342 717 701 330 33 867 138 349 487 398 886 833 529 666 651 403 313 722 546 173 626 141 170 814 248 642 151 839 195 171 626 249 150 154 71 216 297 334 926 585 224 20988
161 480 431 455 727 835 515 217 316 451 542 83 807 412 478 220 96 495 361 616 687 193 333 562 629 770 300 34 41 424 416 835 387 55 519 696 595 66 642 572 444 480 233 288 324 482 704 593 810 650 852
486 554 502 67 672 837 415 44 987 945 65 406 296 726 14 659 951 479 537 185 622 296 18 262 64 432 488 811 851 938 90 501 746 739 325 350 576 734 253 318 834 729 565 289 443 779 994 239 781 991 68887
175 569 805 819 918 756 575 92 880 302 764 823 343 494 648 260 640 756 681 733 816 919 869 524 188 199 856 435 268 116 469 164 669 631 447 657 359 909 371 138 627 251 798 289 488 177 314 831 406 455 71022
621 217 774 177 130 901 587 500 261 850 995 23 390 405 849 995 323 593 465 797 437 66 748 300 822 964 307 587 943 353 573 894 375 369 620 599 748 182 197 765 753 817 478 960 429 513 203 527 592 675 79515
