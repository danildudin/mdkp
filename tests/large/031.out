4028
7
7 11 16 27 33 37 43 
